src/hammer-vlsi/hammer_vlsi/hammer_vlsi_impl.py

```diff
@@ -543,6 +572,13 @@ def specify_all_power_straps_by_tracks(self, layer_names: List[str], ground_net:
             last = layer
         return output
 
+    _power_straps_last_index = -1
+
+    def _power_straps_check_index(self, layer_name: str) -> None:
```

Comment:
What does the underscore convention signify?

Reply:
These are "fake-private"